docker/etna-apache/bin/prepare_shib.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -e
+
+VERIFIED_SHIB_DIR=/opt/shibboleth/
+mkdir -p $VERIFIED_SHIB_DIR
+
+cd $VERIFIED_SHIB_DIR
+curl -o InCommon-metadata.xml https://md.incommon.org/InCommon/InCommon-metadata.xml
+curl -o inc-md-cert.pem https://md.incommon.org/certs/inc-md-cert.pem
+
+# Verify the metadata's signing
+xmlsec1 --verify --pubkey-cert-pem inc-md-cert.pem --id-attr:ID "urn:oasis:names:tc:SAML:2.0:metadata:EntitiesDescriptor" InCommon-metadata.xml
```

Comment:
Xmlsec verifies the metadata signatures before hand.  It seems that this is the step that, for whatever reason, `shibd` takes like forever to accomplish, but `xmlsec` takes like 3 seconds.  The result is that the metadata can be referenced "as is" in the `shibboleth2.xml` file.